eth-contracts/contracts/ForceBridge.sol

```diff
@@ -41,29 +57,193 @@ contract ForceBridge {
     }
 
     modifier onlyAdmin {
-    require(
-            msg.sender == admin,
-            "Only admin can call this function."
-        );
+        require(msg.sender == admin, "Only admin can call this function.");
         _;
     }
 
-    constructor() { admin = msg.sender; }
+    constructor() {
+        admin = msg.sender;
+    }
 
     function changeAdmin(address newAdmin) public onlyAdmin {
         admin = newAdmin;
     }
 
-    function unlock(UnlockRecord[] calldata records) public onlyAdmin {
-        for(uint i=0; i<records.length; i++) {
+    function changeValidators(
+        address[] memory validators,
+        uint256 multisigThreshold,
+        bytes memory signatures
+    ) public {
+        require(
+            validators.length <= VALIDATORS_SIZE_LIMIT,
+            "number of validators exceeds the limit"
+        );
+        require(
+            multisigThreshold <= validators.length,
+            "invalid multisigThreshold"
+        );
+        bytes32 msgHash =
+            keccak256(
+                abi.encodePacked(
+                    "\x19\x01", // solium-disable-line
+                    DOMAIN_SEPARATOR,
+                    keccak256(
+                        abi.encode(
+                            UNLOCK_TYPEHASH,
+                            validators,
+                            multisigThreshold
+                        )
+                    )
+                )
+            );
+
+        uint256 threshold = validators_.length;
+        validatorsApprove(msgHash, signatures, threshold);
+
+        validators_ = validators;
+        multisigThreshold_ = multisigThreshold;
+    }
+
+    function initialize(address[] memory validators, uint256 multisigThreshold)
```

Comment:
`initialize` in force-bridge-eth is used to support upgrade feature. You can just use constructor here.